app/Http/Controllers/VideoController.php

```diff
@@ -75,8 +75,8 @@ public function storeFile(Request $request)
                 ->open($filename)
                 ->export()
                 ->onProgress(function ($percentage) {
-                    $out = new \Symfony\Component\Console\Output\ConsoleOutput();
-                    $out->writeln($percentage . '% transcoded');
+                    //$out = new \Symfony\Component\Console\Output\ConsoleOutput();
```

Comment:
There's no need to keep those debug outputs around if they are commented.
Delete them.